src/cascadia/TerminalSettingsModel/Command.cpp

```diff
@@ -65,10 +68,140 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         return _subcommands ? _subcommands.GetView() : nullptr;
     }
 
+    // Function Description:
+    // - reports if the current command has nested commands
+    // - This CANNOT detect { "name": "foo", "commands": null }

```

Comment:
and why would it? "commands":null _is the very definition of not having nested commands..._